Moviebuff/Views/MoviePosterView.swift

```diff
@@ -0,0 +1,38 @@
+
+import SwiftUI
+
+struct MoviePosterCard: View {
+    
+//    let movie: Movie
+    @ObservedObject var imageLoader = ImageLoader()
+    
+    var body: some View {
+        ZStack {
+            if self.imageLoader.image != nil {
+                Image(uiImage: self.imageLoader.image!)
```

Comment:
Here you are checking for nil on 11 and then force-unwrapping on 12. Instead you should use `if let image = imageLoader.image`, and then use it on line 12 as `image`.